components/checkbox/src/auro-checkbox.js

```diff
@@ -249,6 +250,31 @@ export class AuroCheckbox extends LitElement {
     });
   }
 
+  connectedCallback() {
+    super.connectedCallback();
+
+    this.handleKeyDown = this.handleKeyDown.bind(this);
+    this.addEventListener('keydown', this.handleKeyDown);
+  }
+
+  disconnectedCallback() {
+    super.disconnectedCallback();
+
+    this.removeEventListener('keydown', this.handleKeyDown);
+  }
+
+  /**
+   * @private
+   * @param {*} event
```

Comment:
Should stay consistent and give this param a description. Please take a look at other events passing in an event as parameter in this file.